### DISCUSSION TITLE
How can I pass arguments to searchable in the resource?

### DISCUSSION THREAD

**Discussion by beshoo** [laravel/nova-issues#5683] [Q&A]
- Laravel Version: 10.x
- Nova Version: 4.x
- PHP Version: 8.x
- Database Driver & Version: 8.x mysql
- Operating System and Version: linunx
- Browser type and version: Chroem latest

### Description:
How can I pass arguments to searchable in the resource?

![image](https://github.com/laravel/nova-issues/assets/4635115/2146c950-674e-40a5-b6ef-e4a5c0b062c4)

I need to add role logic to check if the user admin or not..

**Comment by crynobone** [comment]
you can get the current user from `$request->user()`.

**Reply by beshoo** [reply]
![image](https://github.com/laravel/nova-issues/assets/4635115/d112445d-bded-4481-8b84-04047d00d6f9)

**Comment by davidhemphill** [comment]
@beshoo Please create discussions instead of issues. Issues are meant for replicable bugs, not for general application questions.
